packages/oas-utils/src/transforms/import-spec.ts

```diff
@@ -254,77 +254,79 @@ export async function importSpecToWorkspace(
     const pathServers = serverSchema.array().parse(path.servers ?? [])
     servers.push(...pathServers)
 
-    requestMethods.forEach((method) => {
-      const operation: OpenAPIV3_1.OperationObject<{
-        tags?: string[]
-        security?: OpenAPIV3_1.SecurityRequirementObject[]
-      }> = path[method as keyof typeof path]
-      if (operation && typeof operation === 'object') {
-        const operationServers = serverSchema
-          .array()
-          .parse(operation.servers ?? [])
-
-        servers.push(...operationServers)
-
-        // We will save a list of all tags to ensure they exists at the top level
-        // TODO: make sure we add any loose requests with no tags to the collection children
-        operation.tags?.forEach((t) => tagNames.add(t))
-
-        // Remove security here and add it correctly below
-        const { security: operationSecurity, ...operationWithoutSecurity } =
-          operation
-
-        // Grab the security requirements for this operation
-        const securityRequirements = (
-          operationSecurity ??
-          (schema.security as OpenAPIV3_1.SecurityRequirementObject[]) ??
-          []
-        ).flatMap((s) => {
-          const keys = Object.keys(s)
-          if (keys.length) return keys[0]
-          else return []
-        })
-
-        let selectedSecuritySchemeUids: string[] = []
+    // Creates a sorted array of methods based on the path object.
+    const methods = Object.keys(path).filter((method) =>
+      requestMethods.includes(method as any),
+    )
```

Comment:
```suggestion
    const methods = Object.keys(path).filter(isHTTPMethod)
```
we have a type guard for this! you can just move it from the client to oas-utils

Reply:
oooh right right! tysm